pkgs/tools/security/certstrap/default.nix

```diff
@@ -1,21 +1,33 @@
-{ lib, buildGoPackage, fetchFromGitHub }:
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "certstrap";
   version = "1.2.0";
 
-  goPackagePath = "github.com/square/certstrap";
-
   src = fetchFromGitHub {
     owner = "square";
     repo = "certstrap";
     rev = "v${version}";
-    sha256 = "1ymchnn7c9g3pq7rw4lrwsd6z3wfjx90g7qgrw6r5hssl77mnscj";
+    sha256 = "sha256-kmlbz6Faw5INzw+fB1KXjo9vmuaZEp4PvuMldqyFrPo=";
   };
 
+  vendorSha256 = null;
```

Comment:
Null doesn't seem right?

Reply:
It's `null` because the version 1.2.0 has a [vendor](https://github.com/square/certstrap/tree/v1.2.0/vendor) directory. Although, we must start to specify `vendorSha256` manually starting the next release because, apparently, that directory [https://github.com/square/certstrap/commit/92cd0f3331e27f68f04dd6d561f223f773ef8c89](got removed) from the master branch.

Reply:
Ah, interesting, didn't know that